front/app/containers/Admin/projects/all/legacy/Lists/SortableProjectList.tsx

```diff
@@ -31,7 +31,12 @@ const SortableProjectList = ({
   const { mutate: reorderAdminPublication } = useReorderAdminPublication();
   const { formatMessage } = useIntl();
 
-  function handleReorderAdminPublication(itemId: string, newOrder: number) {
+  function handleReorderAdminPublication(
+    itemId: string,
+    newOrderIndex: number
+  ) {
+    if (!adminPublications) return;
+    const newOrder = adminPublications[newOrderIndex].attributes.ordering;
```

Comment:
I guess, before the index would always match the ordering since all admin publications were included. But after the introduction of the community monitor (which I assume has the `ordering: 0`, and is not visible in this list), the attribute was off by one, meaning that the `ordering` would start from 1 instead of 0.